### PR TITLE
Unfinished code for analysis

### PR DESCRIPTION
**Pull request by chctaylor**

### CODE REVIEW

File: substitution.c [hunk 1]
```diff
@@ -1,2 +1,102 @@
...
+        return 1;
+    }
+
+    //Checking for argument input compatibility
+
+    else if (argc == 2)
+    {
+        for (int array_counter = 0, argument = strlen(argv[1]); array_counter < argument; array_counter++)
+        {
+            if (isdigit(argv[1][array_counter]))
```
Comment (crvarner) [inline substitution.c]: what happens if I include a punctuation mark in the key?

File: substitution.c [hunk 2]
```diff
@@ -1,2 +1,102 @@
...
+            }
+        }
+
+        //Checking for duplicate letters
+
+        string argument_input = argv[1];
+
+        //Create placeholder for a copy of user input
+
+        char copied_input[27];
```
Comment (crvarner) [inline substitution.c:40]: I see you leaving room for the null terminator

File: substitution.c [hunk 3]
```diff
@@ -1,2 +1,102 @@
...
+        int duplicate_counter = 0;
+
+        for (int i = 0; i < strlen(copied_input); i++)
+        {
+
+            for (int j = i + 1; j < strlen(copied_input); j++)
+            {
+                if (copied_input[i] == copied_input[j])
+                {
+                    duplicate_counter++;
```
Comment (crvarner) [inline substitution.c:65]: since you are returning immediately upon finding a duplicate, this variable isn't actually used. not hurting anything though.

**Review by crvarner — CHANGES_REQUESTED** [review]

**Review by crvarner — APPROVED** [review]
Looks hot like sauce

**Comment by chctaylor** [comment]
100% per check50

**Pull request merged**